editoast/editoast_derive/src/model.rs

```diff
@@ -204,20 +204,6 @@ fn update_functions(config: &Config) -> TokenStream {
     let table = &config.table;
 
     let model_name = &config.model_name;
-    let documentation = format!(
```

Comment:
Why did the documentation disappear? Is that the culprit in making `cargo chef` not working properly with `cargo chef`?

Reply:
I removed an unused function of the trait `Model::Update`. This was the documentation for the removed function. I removed it since it's part of ModelV1 and should be drop soon.